tests/bliss_index_tests.h

```diff
@@ -38,7 +41,9 @@ class BlissIndexTest : public testing::Test {
             data.push_back(i);
         }
         if (!sorted) {
-            std::random_shuffle(data.begin(), data.end());
+            std::random_device rd;
```

Comment:
What was the reasoning from switching to `std::random_shuffle` to `std::shuffle` with a random generator here?